common/persistence/serialization/thrift_mapper_test.go

```diff
@@ -214,6 +214,8 @@ func TestWorkflowExecutionInfo(t *testing.T) {
 		Memo:                               map[string][]byte{"key_1": []byte("Memo")},
 		VersionHistories:                   []byte("VersionHistories"),
 		VersionHistoriesEncoding:           "VersionHistoriesEncoding",
+		FirstExecutionRunID:                UUID(uuid.New()),
+		PartitionConfig:                    map[string]string{"zone": "dca1"},
```

Comment:
do you mind adding a mapper test where this is nil too?

Reply:
It's already coverred.